sdk/ml/azure-ai-ml/azure/ai/ml/operations/_registry_operations.py

```diff
@@ -140,14 +140,16 @@ def begin_create(
 
     @experimental
     # @monitor_with_activity(logger, "Registry.Delete", ActivityType.PUBLICAPI)
-    def delete(self, *, name: str, **kwargs: Dict) -> None:
+    def delete(self, *, name: str, **kwargs: Dict) -> LROPoller[Registry]:
```

Comment:
Was the previous delete synchronous or asynchronous with no poller?
If the former, this sounds like a breaking change.